SpatialGDK/Documentation/content/game-client-interest-management.md

```diff
@@ -0,0 +1,115 @@
+<%(TOC)%>
+# Game client interest management
+
+The GDK uses [interest]({{urlRoot}}/content/glossary#interest) to control the information that [game clients]({{urlRoot}}/content/glossary#client-workers) receive about the world around them.
+
+Game clients need information about Actors that they don’t own (see the Unreal documentation on [owning](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/OwningConnections) to help them to correctly manipulate the Actors that they _do_ own, and to render the SpatialOS world. Interest enables game clients to receive the relevant information.
+
+You can define your game client’s interest in three ways, which you can use alongside each other:
+
+* **`NetCullDistanceSquared`**<br>
+A property that you define on an Actor’s class. For any Actor of that class or a derived class, game clients whose players are within the distance you specify receive updates about the Actor.<br>
+* **`ActorInterestComponent`**<br>
+An Unreal Actor component that you add to an Actor that a game client might own. This includes Actors that are possessed by Actors that the game client owns. For example, if a game client owns a PlayerController, and the PlayerController possesses a Pawn, then the game client owns the Pawn.
```

Comment:
> This includes Actors that are possessed by Actors that the game client owns.

Just to be pedantic, the only type of Actor that can possess stuff are *PlayerControllers*. And the only type of thing that can be possessed are *Pawns*.

Could this be reworded to be

"This includes the game client's PlayerController, the Pawn the PlayerController is possessing, any Actor the possessed Pawn owns."

Reply:
Great, thanks - this is exactly the kind of background/context I'm missing. 

In the last bit, should it be "any Actors" (plural) or is it a 1:1 relationship between Pawn and Actor as well? I.e. can a Pawn own more than one Actor?

Reply:
Also, I can probably get rid of the final sentence (the example) as it just repeats what you've already said really concisely :)

Reply:
Yup that makes sense.